.gitignore

```
@@ -7,6 +7,7 @@ NOTES
 TODO
 coverage
 dist
+!site/themes/embark/source/js/linkjuice/dist
```

Can we move this line into `site/.gitignore`?

```suggestion
!themes/embark/source/js/linkjuice/dist
```